CHANGELOG.md

```diff
@@ -4,24 +4,32 @@
 
 Changes that are currently in development and have not been released yet.
 
+
+
+## [0.15.3](https://github.com/cossacklabs/themis/releases/tag/0.15.3), April 30 2024
+
+### Swift, Objective-C wrappers
+- Added the '-y' option for the zip program when archiving themis.xcframework to follow symlinks.
```

Comment:
```suggestion
- Added the '-y' option for the zip program when archiving `themis.xcframework` to follow symlinks. It will fix macOS distribution with Themis added via SPM.
```